src/main/java/uk/gov/justice/digital/hmpps/cmd/api/dto/NotificationDto.kt

```diff
@@ -25,17 +25,7 @@ data class NotificationDto @JsonCreator constructor(
 
     companion object {
 
-        fun from(shiftNotifications: Collection<ShiftNotification>, clock: Clock): Collection<NotificationDto> {
-            return shiftNotifications.map {
-                from(it, clock)
-            }
-        }
-
-        fun from(shiftNotification: ShiftNotification, clock: Clock): NotificationDto {
-            val description = NotificationDescription.getNotificationDescription(
-                    shiftNotification,
-                    CommunicationPreference.NONE,
-                    clock)
+        fun from(shiftNotification: ShiftNotification, description: String): NotificationDto {
             return NotificationDto(description,
```

Comment:
by removing the clock, we can also remove the NotificationDescription import in the DTO too. and generate the description in the service layer where the business logic should live.